components/Payment/interfaces/model_interfaces.rst

```diff
@@ -0,0 +1,135 @@
+Model Interfaces
+================
+
+Credit Card Interface
+---------------------
+
+This interface will ask you to implement the following methods:
+
++--------------------------+----------------------------------------+---------------+
+| Method                   | Description                            | Returned type |
++==========================+========================================+===============+
+| getToken()               | Get credit card's token                | string        |
```

Comment:
I think it will be proper to start getters description with ``Returns`` rather than ``Gets``.